docs/features/websockets.rst

```diff
@@ -58,24 +81,53 @@ Then in your ocelot.json add the following to proxy a Route using SignalR. Note
 
 .. code-block:: json
 
-   {
-  "Routes": [
-    {
-      "DownstreamPathTemplate": "/{catchAll}",
-      "DownstreamScheme": "ws",
-      "DownstreamHostAndPorts": [
-        {
-          "Host": "localhost",
-          "Port": 50000
-        }
-      ],
-      "UpstreamPathTemplate": "/gateway/{catchAll}",
-      "UpstreamHttpMethod": [ "GET", "POST", "PUT", "DELETE", "OPTIONS" ]
-    }
- ]
-}
-
-With this configuration set Ocelot will match any SignalR traffic that comes in on / and proxy it to localhost:5001/ws. To make this clearer Ocelot will receive messages from the upstream client, proxy these to the downstream service, receive messages from the downstream service and proxy these to the upstream client.
+     {
+       "Routes": [
+         {
+           "DownstreamPathTemplate": "/{catchAll}",
+           "DownstreamScheme": "ws",
+           "DownstreamHostAndPorts": [
+             {
+               "Host": "localhost",
+               "Port": 50000
+             }
+           ],
+           "UpstreamPathTemplate": "/gateway/{catchAll}",
+           "UpstreamHttpMethod": [ "GET", "POST", "PUT", "DELETE", "OPTIONS" ]
+         }
+       ]
+     }
+
+With this configuration set Ocelot will match any SignalR traffic that comes in on ``/`` and proxy it to ``localhost:5001/ws``.
+To make this clearer Ocelot will receive messages from the upstream client, proxy these to the downstream service, receive messages from the downstream service and proxy these to the upstream client.
+
+SignalR vs WebSocket Secure
+"""""""""""""""""""""""""""
+
+If you are using secured web sockets, use the "**wss**" scheme.
+
+.. code-block:: json
+
+     {
+       "Routes": [
+         {
+           "DownstreamPathTemplate": "/{catchAll}",
+           "DownstreamScheme": "wss",
+           "DownstreamHostAndPorts": [
+             {
+               "Host": "localhost",
+               "Port": 50000
+             }
+           ],
+           "UpstreamPathTemplate": "/gateway/{catchAll}",
+           "UpstreamHttpMethod": [ "GET", "POST", "PUT", "DELETE", "OPTIONS" ]
+         }
+       ]
+     }

```

Comment:
I'm not sure it should be totally copied and pasted here from the code block above!
We could show property change by one line: `"DownstreamScheme": "wss",`